core/src/org/pentaho/platform/engine/core/solution/SimpleParameterProvider.java

```diff
@@ -145,4 +146,36 @@ public void copyAndConvertAdditionalParameters( final Map paramMap ) {
       copyAndConvertParameters( additionalParms );
     }
   }
+
+  /**
+   * Lets Avoid Xss Attacks. It is sanitize incoming value
+   * 
+   * @param value
+   * */
+  private String stripXSS( String value ) {
```

Comment:
Hey Denis,

Please consult with your team on this - there has been a spike on using a known library for doing cleansing as opposed to us rolling our own. I'd rather us use a dedicated JS library for this instead of having various cleanser fragments sprinkled around our various UIs and ppps.